Staging_Dev/20190215T061300Z_remove_temp_stronghold_teleport.sql

```diff
@@ -0,0 +1,6 @@
+USE [perpetuumsa]
+GO
+
+delete from dbo.teleportdescriptions where sourcezone = 16 and targetzone = 0;
```

Comment:
The query here should be more strict as the intention is to delete one particular teleport - not all.
I know technically we want to remove ALL from this zone - but Deletes especially are dangerous to do with non-strict queries.

When in doubt - be specific.
In this case use in addition to the source/target other identifying properties of the teleport.

Also - Referencing other columns by ID is also problematic.
The source and targetzone IDs should be pulled from a query like:
`sourcezone = (SELECT id FROM dbo.zones WHERE name='-zonename-' AND note='-localizedzonename-')...`

Reply:
Ok, I will be more specific for the next times. I'm updating this query at the moment to fix the issue.